tools/interactive/interactivetool_metashrimps.xml

```diff
@@ -10,7 +10,7 @@
         </entry_point>
     </entry_points>
     <command><![CDATA[
-
+        sed -i 's?input\$file\$datapath?"'$inputfile'"?g' /srv/shiny-server/sample-apps/META/App_metadig/app.R &&
```

Comment:
Oh gosh, what is that :)

Reply:
are you sure `$file` is not treaded as cheetah variable?

Reply:
can you ship your own app.R?

Reply:
> 

It is workin gproperly on my local planemo instance, so apparently, not ;)

Reply:
Ok, we can try ... :)